packages/wallet-specs/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "wallet-protocols",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "generateConfigs": "yarn run ts-node bin/generateConfigs.ts && npx prettier --write src/protocols/*/protocol.config.js",
+    "test": "jest"
+  },
+  "dependencies": {
+    "xstate": "^4.7"
+  },
+  "devDependencies": {
+    "@types/jest": "^24.0.23",
+    "@types/node": "^12.12.7",
+    "fs": "^0.0.1-security",
+    "i": "^0.3.6",
+    "jest": "^24.9.0",
+    "madge": "^3.6.0",
```

Comment:
This was installed to detect circular dependencies; it's not really needed as a dev dependency.
```suggestion
```